shared/people/index.shared.tsx

```diff
@@ -57,23 +59,21 @@ export const itemToComponent: (item: Types.PeopleScreenItem, props: Props) => Re
   return null
 }
 
-const EmailVerificationBanner = ({email, clearJustSignedUpEmail}) => {
+const EmailVerificationBanner = ({email}) => {
+  const dispatch = Container.useDispatch()
+  React.useEffect(() => () =>
+    // Only have a cleanup function
+    dispatch(SignupGen.createClearJustSignedUpEmail())
```

Comment:
This cleanup function fires on every render, if you want it to run on unmount add the second param to `useEffect`.

https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects